page.go

```diff
@@ -403,7 +403,23 @@ type gstate struct {
 
 // Content returns the page's content.
 func (p Page) Content() Content {
-	strm := p.V.Key("Contents")
+	switch v := p.V.Key("Contents"); v.Kind() {
+	case Stream:
+		return p.contentForStream(v)
+	case Array:
+		var c Content
+		for i := 0; i < v.Len(); i++ {
+			cfs := p.contentForStream(v.Index(i))
+			c.Text = append(c.Text, cfs.Text...)
+			c.Rect = append(c.Rect, cfs.Rect...)
+		}
+		return c
+	default:
+		panic("bad content kind")
```

Comment:
Good question @josharian: I am not sure, I am not well versed with the PDF spec but I thought it was a little unexpected to panic on other types. So far I've only seen mentions of Stream and Array for "Content" in the spec.
Perhaps: I think it would handle the "other cases" properly because it contains the almost the same code before this patch, and that has worked in the past.